src/main/reactjs/src/pages/ClerkSendEmailPage.tsx

```diff
@@ -166,6 +166,16 @@ export const ClerkSendEmailPage = () => {
     handleFieldError('message')(event);
   };
 
+  const getEmailBodyHelperText = () => {
+    const errorToShow =
+      fieldErrors.message.length > 0 ? `${fieldErrors.message}.` : '';
+    const maxLength = Utils.maxTextAreaLength();
+
+    return `${errorToShow} ${email.body.length} / ${maxLength} ${t(
+      'component.contactRequestForm.characters'
```

Comment:
Yhteinen lokalisaatio id. @kimmotaskinen, sä taisit näitä omassa taskissa yhdistellä?